src/components/MediaPlayer/VideoJS/VideoJSPlayer.js

```diff
@@ -527,6 +527,20 @@ function VideoJSPlayer({
       if (IS_SAFARI) {
         handleTimeUpdate();
       }
+
+      /**
+       * When either player/browser tab is muted Safari and Chrome in iOS doesn't seem to 
+       * load enough data related to audio-only media for the Video.js instance to play 
+       * on page load.
+       * Since, it is not possible to detect muted tabs in JS the condition avoids
+       * checking for muted state altogether.
+       * Without this, Safari will not reach player.readyState() = 4, the state
+       * which indicates the player that enough data is available on the media
+       * for playback.
+       */
+      if (!isVideo && (IS_SAFARI || IS_IOS) && player.readyState() != 4) {
```

Comment:
Are there more ready states than 4?  If so should this be `player.readyState() < 4`?

Reply:
It looks like ready states only go to 4.
https://docs.videojs.com/player#readyState

Reply:
Yes, the media would not play unless it reaches 4.